src/components/Button.tsx

```diff
@@ -210,13 +237,7 @@ const getStyles = (theme: Theme) => {
         backgroundColor: primaryColor,
       },
     }),
-    invertedDisabled: StyleSheet.create({
-      // eslint-disable-next-line react-native/no-unused-styles
-      container: {
-        backgroundColor: disabledColor,
-        borderColor: disabledColor,
-      },
-    }),
+    invertedDisabled: StyleSheet.create({}),
```

Comment:
This seems different from previous code. Can we avoid it? If not, why was it needed?